### PR TITLE
Setup/initialize

### PR DESCRIPTION
**Pull request by M-AminAlizadeh**
The things that I did in this PR:

1. Use react-router v6 for routing in 2 pages (Home and Categories)
2. No linter errors
3. Build component initial structure

### CODE REVIEW

File: src/components/Books.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import Book from './Book';
+
+const Books = () => (
+  <>
+    <Book />
+    <Book />
+    <Book />
```
Comment (aamirkhan2478) [inline src/components/Books.js:7]: - [ ] Please takes the book data as a prop and render the `title` and the `author` check below :point_down: my suggestion :slightly_smiling_face: 

```suggestion
      <Book title="The Pilgrim's Progress" author="John Bunyan" />
      <Book title="Robinson Crusoe" author="Daniel Defoe" />
```

File: src/components/Book.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+const Book = () => (
+  <div className="book-container">
+    <span className="book-title">Book title</span>
+    <img src="https://img.icons8.com/ios/20/trash--v1.png" alt="trash--v1" />
+  </div>
+);
```
Comment (aamirkhan2478) [inline src/components/Book.js:6]: - [ ] As I mentioned above :point_up_2: pass `title` and `author` props inside the `Book` function then you will able to use these props in your `Book` component and use these props in your `spam` tag :slightly_smiling_face: 
- [ ] Please remove the `img` tag and replace it with the `button` tag with `Remove` text. :slightly_smiling_face: 

**Check below :point_down: my suggestions**
```suggestion
const Book = ({title, author}) => (
  <div className="book-container">
    <span className="book-title">{title}</span>
    <span className="book-title">{author}</span>
    <button>Remove</button>
  </div>
);
```

File: src/App.js
Comment (aamirkhan2478) [inline src/App.js:1]: - [ ] I noticed that you use the `Navbar` component inside the `Home page` and `Categories page` which are not correct please add your  `Navbar` component inside the start of the `App` component then you will see `Navbar` component on both `Home page` and `Categories page` so kindly remove `Navbar` component from `Home page` and `Categories page` and add `Navbar` component inside the start of the `App` component :+1: check below :point_down: my suggestions :slightly_smiling_face:

```JSX
<BrowserRouter>
<Navbar />
      <Routes>
        <Route path="/" element={<Home />} />
        <Route path="/categories" element={<Categories />} />
      </Routes>
</BrowserRouter>
```

**Review by aamirkhan2478 — CHANGES_REQUESTED** [review]
Hi @M-AminAlizadeh , 



Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

 ## Highlights
✔️ Good Effort
✔️ Your design is good.
✔️ All linters passed

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by t-yanick — APPROVED** [review]
# STATUS: MILESTONE APPROVED :1st_place_medal: 

Hi @M-AminAlizadeh , 

### Great job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Great job implementing changes requested by the previous CR

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**